examples/webgpu_audio_processing.html

```diff
@@ -59,9 +59,9 @@
 
 				// compute audio
 
-				renderer.compute( computeNode );
-
-				const waveArray = new Float32Array( await renderer.getArrayBuffer( waveGPUBuffer ) );
+				await renderer.compute( computeNode );
+				await renderer.populateArray( waveGPUBuffer );
```

Comment:
I think `populateArray` a bit generic term for this?

Reply:
Yeah... Maybe something like `deuploadBufferAttribute` or `unuploadBufferAttribute`?